markdown/stories/invalid_data.md

```diff
@@ -11,7 +11,9 @@ Babylon should stop execution and notify the user.
 
 Test | Test Name | Count
 -----|-----------|-------
-data_test.go| <ul><li>TestHasValidPathForCTL</li><li>TestHasInvalidDataPath</li><li>TestHasValidComplexPathCTLAndMod</li></ul> |3
+data_test.go| TestHasValidPathForCTL | 1
+data_test.go| TestHasInvalidDataPath | 1
```

Comment:
Previously we tried the multiline approach, but this was so unpredictable in how it would render down in pandoc that I decided to move to having individual lines per test.